pinot-common/src/main/java/org/apache/pinot/common/response/broker/BrokerResponseNative.java

```diff
@@ -127,7 +127,7 @@ private static BrokerResponseNative getBrokerResponseExplainPlanOutput() {
   /**
    * Get a new empty {@link BrokerResponseNative}.
    */
-  public static BrokerResponseNative empty() {
+  private static BrokerResponseNative empty() {
     return new BrokerResponseNative();
```

Comment:
Does it make sense to just return the singleton instance here?

Reply:
I see your point. It seems there are quite a few fields that need to be updated. That doesn't make sense to just return an instance here. Let me think of some other ways to do it. Thanks!